Fabric.Authorization.API/Bootstrapper.cs

```diff
@@ -107,6 +107,13 @@ protected override void ApplicationStartup(TinyIoCContainer container, IPipeline
             ConfigureApplicationRegistrations(container);
             var dbBootstrapper = container.Resolve<IDbBootstrapper>();
             dbBootstrapper.Setup();
+
+            var appConfig = container.Resolve<IAppConfiguration>();
+            if (appConfig.MigrateDuplicateGroups)
+            {
+                var groupMigratorService = container.Resolve<GroupMigratorService>();
+                var migrationResults = await groupMigratorService.MigrateDuplicateGroups();

```

Comment:
What do we do with this result?

Reply:
Nothing yet - I was thinking of potentially logging the entire structure out.